bodo/pandas/_physical_conv.cpp

```diff
@@ -112,6 +114,31 @@ std::shared_ptr<PhysicalExpression> buildPhysicalExprTree(
                     buildPhysicalExprTree(bce->children[0]),
                     buildPhysicalExprTree(bce->children[1]), expr_type));
         } break;  // suppress wrong fallthrough error
+        case duckdb::ExpressionClass::BOUND_OPERATOR: {
+            // Convert the base duckdb::Expression node to its actual derived
+            // type.
+            duckdb::unique_ptr<duckdb::BoundOperatorExpression> bce =
+                dynamic_cast_unique_ptr<duckdb::BoundOperatorExpression>(
+                    std::move(expr));
+            switch (bce->children.size()) {
+                case 1: {
+                    return std::static_pointer_cast<PhysicalExpression>(
+                        std::make_shared<PhysicalUnaryExpression>(
+                            buildPhysicalExprTree(bce->children[0]),
+                            expr_type));
+                } break;
+                case 2: {
+                    return std::static_pointer_cast<PhysicalExpression>(
```

Comment:
Can `BOUND_OPERATOR` have multiple children? It's not just unary operators? What does it mean?

Reply:
It can.  I thought that arithmetic went through here with 2 children but there are definitely cases clear from the code where it is more than one.  It isn't super clear what those other cases are yet so this is a bit of future infrastrucutre.